setup.cfg

```diff
@@ -80,7 +91,7 @@ testpaths = "sunpy" "docs"
 norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "astropy_helpers" "examples" "sunpy[\/]data"
 doctest_plus = enabled
 doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
-addopts = --doctest-rst -m "not figure"
+addopts = -p no:warnings --doctest-rst -m "not figure"
```

Comment:
you adding this back on purpose?

Reply:
Yes. There are warnings not caused by sunpy in the log and scrolling pasting them (and the sunpy ones) on the CI over and over again was driving me mad.